torchtext/legacy/README.rst

```diff
@@ -0,0 +1,53 @@
+Legacy
+======
+
+In v0.9.0 release, we move the following legacy code to `torchtext.legacy <#legacy>`_. This is part of the work to revamp the torchtext library and the motivation has been discussed in `Issue #664 <https://github.com/pytorch/text/issues/664>`_:
+
+* ``torchtext.legacy.data.field``
+* ``torchtext.legacy.data.batch``
+* ``torchtext.legacy.data.example``
+* ``torchtext.legacy.data.iterator``
+* ``torchtext.legacy.data.pipeline``
+* ``torchtext.legacy.datasets``
+
+We have a `migration tutorial <https://fburl.com/9hbq843y>`_ to help users switch to the torchtext datasets in ``v0.9.0`` release. For the users who still want the legacy components, they can add ``legacy`` to the import path.
```

Comment:
fburls are not public

Reply:
You can export a colab notebook as an ipython notebook and upload the ipython notebook here.

Then you can create a colab link that uses the ipython on github like so:
```
https://colab.research.google.com/github/pytorch/nestedtensor/blob/master/tutorials/notebooks/basic.ipynb
```

Reply:
Move the tutorial to `examples/migration_tutorial` folder https://colab.research.google.com/github/zhangguanheng66/text/blob/migration_tutorial/examples/legacy_tutorial/migration_tutorial.ipynb